tests/drivers/regulator/fixed/testcase.yaml

```diff
@@ -3,7 +3,8 @@ tests:
     tags: drivers regulator
     harness: ztest
     harness_config:
-      fixture: regulator_gpio_loopback
+      fixture: gpio_loopback
+    depends_on: gpio
     platform_allow: nrf52840dk_nrf52840
```

Comment:
platform_allow may also be deleted

Reply:
Due to problems on the `native_posix` this list should exist.